HW_5/task26.py

```diff
@@ -0,0 +1,13 @@
+# Задача 26: Напишите программу, которая на вход принимает два числа A и B, 
+#     и возводит число А в целую степень B с помощью рекурсии.
+# A = 3; B = 5 -> 243 (3^5) 
+# A = 2; B = 3 -> 8
+
+a = int(input("Введите первое число a: "))
+b = int(input("Введите второе число b: "))
+
+def power(c,d):
+    if d == 0: return 1
+    else: return c*power(c,d-1)
```

Comment:
Такая запись хоть и допустима, но не принята. А еще блок кода else не нужен, так как return завершает работу функции
```suggestion
    if d == 0:
        return 1
    return c * power(c, d - 1)
```